src/V3Options.h

```diff
@@ -450,6 +451,7 @@ class V3Options final {
     bool decoration() const { return m_decoration; }
     bool dpiHdrOnly() const { return m_dpiHdrOnly; }
     bool dumpDefines() const { return m_dumpLevel.count("defines") && m_dumpLevel.at("defines"); }
+    bool dumpTreeDot() const { return m_dumpTreeDot; }
```

Comment:
```suggestion
    bool dumpDefines() const { return m_dumpLevel.count("tree-dot") && m_dumpLevel.at("tree-dot"); }
```

Then remove the option parsing from V3Options.cpp. I changed the dump options here recently.